fishtest/fishtest/views.py

```diff
@@ -1137,15 +1137,11 @@ def tests(request):
           run['results']['wins'] + run['results']['draws']
           + run['results']['losses']))
 
-      games_per_minute = 0.0
       machines = request.rundb.get_machines()
+      games = request.gendb.get('gamesper64s')
```

Comment:
What happens when this `get('gamesper64s')` fails (returns `None`). You should map that error to `games_per_minute = 0.0`

Reply:
Good point.